1.10/cli/command-reference/dcos-auth/dcos-auth-login.md

```diff
@@ -4,7 +4,7 @@ menu_order: 2
 ---
 
 # Description
-Log in to DC/OS authentication. 
```

Comment:
I would have preferred one of your alternatives. This was the original wording and I left it because I though maybe I was missing something..